### DISCUSSION TITLE
No ASRock UEFI in repository

### DISCUSSION THREAD

**Discussion by mbledkowski** [fwupd/fwupd#5805] [Ideas]
Hello,
Why is there no ASRock UEFI in the fwupd repository? I would like to have the ability to update my MOBO firmware through Gnome Firmware.
Best regards
Maciej Błędkowski

**Comment by superm1** [comment]
Firmware uploads are controlled by manufacturers.  If a manufacturer or product isn't present you'll need to contact that manufacturer to ask them to add it.

So in this case you'll need to contact ASRock to request them to add firmware for your system.
